packages/chrony/build.sh

```diff
@@ -0,0 +1,21 @@
+TERMUX_PKG_HOMEPAGE=https://chrony-project.org/
+TERMUX_PKG_DESCRIPTION="chrony is an implementation of the Network Time Protocol (NTP)"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="@marek22k"
+TERMUX_PKG_VERSION=4.4
+TERMUX_PKG_SRCURL=https://chrony-project.org/releases/chrony-${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=eafb07e6daf92b142200f478856dfed6efc9ea2d146eeded5edcb09b93127088
+TERMUX_PKG_DEPENDS="pkg-config readline libtomcrypt libnettle libnss libgnutls libcap"
+
+termux_step_configure() {
+    # chrony can currently (not yet) adjust the time on Android
+    # see https://gitlab.com/chrony/chrony/-/issues/1
+    # For the build to be successful, this functionality must be deactivated with `--disable-refclock`.
+
+    "$TERMUX_PKG_SRCDIR/configure" \
+        --disable-refclock \
+        --prefix="${TERMUX_PREFIX}" \
+        --sysconfdir="${TERMUX_PREFIX}/etc" \
+        --localstatedir="${TERMUX_PREFIX}/var" \
+        --host-system="${TERMUX_ARCH}"
+}
```

Comment:
I'm a little unsure about that. I looked at the configuration of various packages and thought that this would be a good fit.

The CI has indicated that the options that Termux provides ./configure by default are not recognized. To avoid the warnings I thought it would be good not to include the unrecognized options. I have tried to provide the most suitable options.

Reply:
Thanks for the explanation! I have now adapted it.